.lintstagedrc

```diff
@@ -0,0 +1,3 @@
+{
+  "*.js": ["eslint", "prettier --write"]
```

Comment:
Let's add `.ts` files for the future. Also, what about `.vue` files? I see this extension in the lint script.

Reply:
add checks for `.ts` and `.vue`